tests/data/mlip_fine_tune.yml

```diff
@@ -0,0 +1,34 @@
+name: test-finetuned
+foundation_model: "tests/models/mace_mp_small.model"
+model: ScaleShiftMACE
+loss: universal
+train_file: "./tests/data/mlip_fine_tune_train.xyz"
+valid_file: "./tests/data/mlip_fine_tune_train.xyz"
+test_file: "./tests/data/mlip_test.xyz"
+E0s: foundation
+valid_fraction: 0.05
+energy_weight: 1.0
+forces_weight: 10.0
+stress_weight: 100.0
+stress_key: dft_stress
+energy_key: dft_energy
+forces_key: dft_forces
+compute_stress: True
+compute_forces: True
+clip_grad: 100
+error_table: PerAtomRMSE
+lr: 0.005
+scaling: rms_forces_scaling
+batch_size: 4
+max_num_epochs: 1
+ema: True
+ema_decay: 0.995
+amsgrad: True
+default_dtype: float64
```

Comment:
make float32
```suggestion
default_dtype: float32
```

Reply:
I get a type mismatch:

```
  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/mace/cli/run_train.py", line 686, in run
    tools.train(

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/mace/tools/train.py", line 148, in train
    valid_loss, eval_metrics = evaluate(
                               ^^^^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/mace/tools/train.py", line 357, in evaluate
    output = model(
             ^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/mace/modules/models.py", line 340, in forward
    node_feats = self.node_embedding(data["node_attrs"])
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/mace/modules/blocks.py", line 44, in forward
    return self.linear(node_attrs)
           ^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/e3nn/o3/_linear.py", line 280, in forward
    return self._compiled_main(features, weight, bias)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

RuntimeError: The following operation failed in the TorchScript interpreter.
Traceback of TorchScript (most recent call last):
  File "<eval_with_key>.26", line 15, in forward
    reshape_3 = reshape_1.reshape((4, 128));  reshape_1 = None
    mul = reshape_2 * 0.5;  reshape_2 = None
    tensordot = torch.functional.tensordot(mul, reshape_3, ([1], [0]), out = None);  mul = reshape_3 = None
                ~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
    permute = tensordot.permute([0, 2, 1]);  tensordot = None
    reshape_4 = permute.reshape(getitem_1, 128);  permute = getitem_1 = None
  File "/home/ek/PSDI/venv_janus_11_new/lib/python3.11/site-packages/torch/functional.py", line 1213, in tensordot

    if out is None:
        return _VF.tensordot(a, b, dims_a, dims_b)  # type: ignore[attr-defined]
               ~~~~~~~~~~~~~ <--- HERE
    else:
        return _VF.tensordot(a, b, dims_a, dims_b, out=out)  # type: ignore[attr-defined]
RuntimeError: both inputs should have same dtype
```

Reply:
ok leave it to 64 I suspect the issue is of mixing model precision